src/button/index.d.ts

```diff
@@ -59,10 +59,9 @@ export const StyledEndEnhancer: StyletronComponent<any>;
 export const StyledLoadingSpinner: StyletronComponent<any>;
 export const StyledLoadingSpinnerContainer: StyletronComponent<any>;
 
-export class Button extends React.Component<ButtonProps> {
-  internalOnClick(...args: any): void;
```

Comment:
There's no way you can access `internalOnClick` because the ref is being forwarded to the html button element not the class